README.md

```diff
@@ -96,45 +91,37 @@ defmodule Interleave.Pipeline do
   alias Membrane.File.{Sink, Source}
 
   @impl true
-  def handle_init({path_to_wav_1, path_to_wav_2}) do
-    children = %{
-      file_1: %Source{location: path_to_wav_1},
-      file_2: %Source{location: path_to_wav_2},
-      parser_1: Membrane.WAV.Parser,
-      parser_2: Membrane.WAV.Parser,
-      interleaver: %Membrane.AudioInterleaver{
-        input_caps: %Membrane.RawAudio{
+  def handle_init(_ctx, {path_to_wav_1, path_to_wav_2}) do
+    structure = [
+      child({:file, 1}, %Source{location: path_to_wav_1})
+      |> child({:parser, 1}, Membrane.WAV.Parser)
+      |> get_child(:interleaver),
+
+      child({:file, 2}, %Source{location: path_to_wav_2})
+      |> child({:parser, 2}, Membrane.WAV.Parser)
+      |> get_child(:interleaver),
+
+
+      child(:interleaver, %Membrane.AudioInterleaver{
+        input_stream_format: %Membrane.RawAudio{
           channels: 1,
           sample_rate: 16_000,
           sample_format: :s16le
         },
         order: [:left, :right]
-      },
-      file_sink: %Sink{location: "output.raw"}
-    }
-
-    links = [
-      link(:file_1)
-      |> to(:parser_1)
-      |> via_in(Pad.ref(:input, :left))
-      |> to(:interleaver),
-      link(:file_2)
-      |> to(:parser_2)
-      |> via_in(Pad.ref(:input, :right))
-      |> to(:interleaver),
-      link(:interleaver)
-      |> to(:file_sink)
+      })
+      |> child(:file_sink, %Sink{location: "output.raw"})
     ]
 
-    {{:ok, spec: %ParentSpec{children: children, links: links}}, %{}}
+    {[spec: structure], %{}}
   end
 end
 
 ```
 
 ## Copyright and License
 
-Copyright 2021, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane)
+Copyright 2023, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane)
```

Comment:
You shouldn't change that - it indicates the year copyrights were claimed, i.e. year of the first publish
```suggestion
Copyright 2021, [Software Mansion](https://swmansion.com/?utm_source=git&utm_medium=readme&utm_campaign=membrane)
```